components/raftstore-v2/src/operation/ready/async_writer.rs

```diff
@@ -0,0 +1,199 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
```

Comment:
for async_write, what's the key difference between v1 and v2? (except the difference due to new structure names)

Reply:
There will be no writes to kv engine in v2.